rinja_derive/src/generator/node.rs

```diff
@@ -1167,7 +1167,8 @@ impl<'a> Generator<'a, '_> {
                         idx
                     };
                     lines.write(format_args!(
-                        "(&&rinja::filters::Writable(expr{idx})).rinja_write(__rinja_writer)?;",
+                        "(&&&rinja::filters::Writable(expr{idx})).\
```

Comment:
Why adding a new reference here btw?

Reply:
I added the values as new argument to `WriteWritable`. This way it gets automagically passed on to any `{{ template }}`s. To make this work I need to tell `Template`s other `FastWritable`s apart.